src/database/migrations/1739918380436_member_user_public_key.ts

```diff
@@ -0,0 +1,19 @@
+import { type Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+    await db.schema
+        .createTable('member_user_public_key')
+        .addColumn('id', 'uuid', (col) => col.defaultTo(sql`v7uuid()`).primaryKey())
+        .addColumn('user_id', 'uuid', (col) => col.notNull())
```

Comment:
This is a stab at a column name for referencing a user, but I'm not sure if it aligns with the eventual vision :sweat_smile: . (e.g. could this be a FK to a future table? :man_shrugging: )